lab_3_generate_by_ngrams/start.py

```diff
@@ -14,6 +20,24 @@ def main() -> None:
     with open("./assets/Harry_Potter.txt", "r", encoding="utf-8") as text_file:
         text = text_file.read()
     result = None
+    processor = TextProcessor('_')
+    encoded_text = processor.encode(text) or ()
+    print(f"Зашифрованный текст: {encoded_text}")
+    decoded_text = processor.decode(encoded_text) or ()
+    print(f"Расшифрованный текст: {decoded_text}")
+
+    model = NGramLanguageModel(encoded_text, 7)
+    frequency = model.build()
+    print(f"Частотный словарь n-грамм: {frequency}")
+
+    greedy_generator = GreedyTextGenerator(model, processor)
+    greedy_generator_text = greedy_generator.run(51, 'Vernon')
+    print(f"Результат работы жадного генератора: {greedy_generator_text}")
+
+    beam_search_generator = BeamSearchTextGenerator(model, processor, 3)
+    beam_search_text = beam_search_generator.run('Vernon', 56)
+    print(f"Результат работы алгоритма лучевого поиска: {beam_search_text}")
```

Comment:
Anna, I know that you prefer linguistics rather than programming, but unfortunately we are using english language in our course, please change 😸